helm/prometheus/Chart.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v2
+name: prometheus
+description: A Helm chart of Prometheus resources for OLMv1
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.0
```

Comment:
I think we should match the version with the release version and have a GitHub action that automatically opens a PR to update it when we do a release for now. In the future, we will need to bump before release. Do we not plan to distribute the chart instead of the shell?